### ISSUE TITLE
Data_generator function

### ISSUE THREAD

**Issue by desmanda** [qqwweee/keras-yolo3#153]
Hallo,
I am trying to understand the code to debug my problem.
I trained the model for two classes but I am getting loss value around 30.
I would like to know for what `np.zeros(batch_size)` is needed for the data_generator?
( last line of the function` yield [images_data, *y_true], np.zeros(batch_size)` ).
Thank you!
